src/content/docs/accounts/accounts-billing/account-structure/new-relic-account-structure.mdx

```diff
@@ -9,26 +9,43 @@ redirects:
   - /docs/accounts/accounts-billing/new-relic-one-pricing-users/new-relic-account-structure 
 ---
 
-Depending on your [user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models), you have different options for adding and managing accounts and assigning users to them. We have two user models:
+<Callout variant="tip">
+This doc is for organizations with users on our [newer user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models). If your users are still on our original user model, see [Original account access](/docs/accounts/original-accounts-billing/original-users-roles/original-account-structure).
+</Callout>
 
-* [Our newer user model](#new-model)
-* [Our original user model](/docs/accounts/original-accounts-billing/original-users-roles/original-account-structure)
+Understanding the structure of New Relic organizations and accounts will help you with finding data in New Relic and setting up user access. 
 
-## Our newer user model [#new-model]
+## Organizations and accounts [#organization-accounts]
 
-This section is for organizations whose users are on our [newer user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models).
+At New Relic, an **organization** represents a New Relic customer. Your New Relic organization contains everything relevant to you and your team members: your organization's accounts, users, and data.
 
-At New Relic, an "organization" represents a New Relic customer. The organization contains everything relevant to a New Relic customer: its accounts, its users, and its data.
+When you sign up for New Relic, it creates an organization containing a single account. A Standard edition organization can only have a single account. Pro and Enterprise edition organizations can add more accounts. 
 
-When a New Relic organization is created, it contains a single account. A Standard edition organization can only have a single account, but Pro and Enterprise edition organizations can add more accounts. 
+An account can have many sources of data reporting to it. All data reported to New Relic requires an [account ID](/docs/accounts/accounts-billing/account-structure/account-id), which governs where that data goes and where it's stored. That ID is also used for some account-specific tasks, like making API calls. 
```

Comment:
This section is great, but I think it could benefit from one more paragraph that boils it down to a specific example:

``` For example, an org could have a python team and a java team. Each team with their own account under that organization. ```

A diagram could also be really useful here instead